infrastructure/src/main/resources/templates/dozenten/veranstaltung.html

```diff
@@ -108,6 +108,17 @@ <h2>Studenten hinzufügen:</h2>
                         File to upload:<input type="file" name="file" accept=".csv" />
                         <input type="submit" value="Upload" class="btn btn-primary float-right" />
                     </form>
+                    <hr class="col-xs-12">
+                    <form class="form-horizontal was-validated" method="post" th:action="'/feedback/dozenten/event/addStudent/' + ${veranstaltung.veranstaltungsNr}">
+                        <div class="row">
+                            <div class="col-sm-6">
+                                <input type="text" placeholder="Matrikelnummer eingeben" name="newStudent">
```

Comment:
Please put the keyword _required_ into the input tag. Otherwise, you can add a student with empty username.